src/mbgl/renderer/layers/render_raster_layer.cpp

```diff
@@ -239,7 +242,7 @@ void RenderRasterLayer::update(gfx::ShaderRegistry& shaders,
                                               ? gfx::TextureFilterType::Nearest
                                               : gfx::TextureFilterType::Linear;
 
-    auto createBuilder = [&context, &renderPass, this]() -> std::unique_ptr<gfx::DrawableBuilder> {
+    auto createBuilder = [&context, this]() -> std::unique_ptr<gfx::DrawableBuilder> {
```

Comment:
Capture `renderPass` by value here

Reply:
Capturing by value or references produces `Lambda capture 'renderPass' is not required to be captured for this use`, that's why I removed it.

Reply:
MSVC wants an explicit capture: https://godbolt.org/z/MxGTEbsdK

Reply:
I put it back the way it was, Stefan's still working on this anyway.